scheduler-server/Dockerfile

```diff
@@ -11,8 +17,7 @@ ENV PATH="/root/.nvm/versions/node/v${NODE_VERSION}/bin/:${PATH}"
 RUN node --version
 RUN npm --version
 
-COPY build/distributions/*.tar /usr/src/app/server.tar
-RUN cd /usr/src/app && tar --strip-components 1 -xf server.tar
+COPY --from=0 /usr/src/app/extracted /usr/src/app
```

Comment:
If you don't mind making this change I think it'd be nice to see an alias used for `--from`. You can alias a `FROM ... AS <alias>` and then do `--from=<alias>` to avoid relying on an index.

See https://stackoverflow.com/questions/66353510/what-is-from-used-in-copy-command-in-dockerfile.

Reply:
Thanks, I amended that change.

Reply:
Awesome, thanks!